Frontend/src/App.js

```diff
@@ -1,26 +1,43 @@
-import logo from './logo.svg';
+import { useRef, useState } from 'react';
 import './App.css';
 import Canvas from './components/Canvas';
+import Constants from './constants/constants';
 
 function App() {
+  const constants = new Constants();
+  const canvasRef = useRef(null);
+  const [LAPI_resp, setLAPI_Resp] = useState(null);
+  const getLatex = async () => {
+    const imgData = canvasRef.current.toDataURL('image/png');
+    const resp = await fetch(`${constants.SERVER_BASE_URL}/${constants.EP_POST_CANVAS}`, {
+      method: "POST",
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({ imageData: imgData }),
+    });
+    const sresp = await resp.json();
+    setLAPI_Resp(sresp);
+    console.log(LAPI_resp);
+  }
+
   return (
     <div className="App">
       <header className="App-header">
-      <h1>Canvas Drawing</h1>
-        <Canvas />
+        <h1>Canvas Drawing</h1>
+        <Canvas canvasRef={canvasRef} />
```

Comment:
If canvas creation fails somehow, this will give error. Should check before passing it to childrens.